### PR TITLE
Add Number::Bytes::Human to ecosystem. 

### PR DESCRIPTION
**Pull request by dugword**
See https://github.com/dugword/Number-Bytes-Human

### CODE REVIEW

**Comment by astj** [comment]
@dugword Hi. I found your module depends on `Test::META` on test, but is not declared in META.info therefore `panda install` fails!

```
$ panda install  git://github.com/dugword/Number-Bytes-Human.git
==> Fetching Number::Bytes::Human
==> Building Number::Bytes::Human
==> Testing Number::Bytes::Human
t/010-basic.t ...... ok
t/020-functions.t .. ok
t/030-class.t ...... ok
===SORRY!===
Could not find Test::META at line 7 in:
    /Users/astj/.panda-work/1482827476_1/lib
    /Users/astj/.perl6
    /Users/astj/.rakudobrew/moar-nom/install/share/perl6/site
    /Users/astj/.rakudobrew/moar-nom/install/share/perl6/vendor
    /Users/astj/.rakudobrew/moar-nom/install/share/perl6
    CompUnit::Repository::AbsolutePath<140197825517920>
    CompUnit::Repository::NQP<140197825572440>
    CompUnit::Repository::Perl5<140197825572480>
t/040-my-meta.t ....
Dubious, test returned 1 (wstat 256, 0x100)
No subtests run

Test Summary Report
-------------------
t/040-my-meta.t  (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: No plan found in TAP output
Files=4, Tests=46,  2 wallclock secs ( 0.03 usr  0.01 sys +  1.27 cusr  0.20 csys =  1.51 CPU)
Result: FAIL
```

**Comment by astj** [comment]
And as mentioned in https://github.com/perl6/ecosystem/pull/274#issuecomment-267628573 , now `META6.json` is preferred metadata filename (though `META.info` is still supported).

**Comment by jonathanstowe** [comment]
You may want to adjust your meta test to something like:

```perl6
use Test;

my Bool $got-test-meta = True;

my &m-meta-ok;

BEGIN {
    require Test::META <&meta-ok>;
    $got-test-meta = True;

    &m-meta-ok = &meta-ok;

    CATCH {
        when X::CompUnit::UnsatisfiedDependency {
            plan 1;
            skip-rest "no Test::META - skipping";
            done-testing;
            exit;
        }
    }

}

plan 1;

if $got-test-meta {
    m-meta-ok();
}
else {
    skip "no Test::META skipping";
}


done-testing;
```

If you don't want to add the Test::META as a dependency.

**Comment by dugword** [comment]
Thank you so much for the kind feedback.

I have renamed META.info to META6.json, and re-wrote the t/040-my-meta.t script to skip the test when Test::META isn't installed.

The package was tested with:
` panda install  git://github.com/dugword/Number-Bytes-Human.git` (Didn't know panda install straight from github, thanks!)

With both Test::META installed and uninstalled. Works both ways now.

File: META.list [hunk 1]
```diff
@@ -758,3 +758,4 @@ https://raw.githubusercontent.com/samcv/IRC-TextColor/master/META6.json
 https://raw.githubusercontent.com/jonathanstowe/Tinky-JSON/master/META6.json
 https://raw.githubusercontent.com/melezhik/sparrowdo-chef-client/master/META.info
 https://raw.githubusercontent.com/jnthn/p6-test-scheduler/master/META6.json
+https://raw.githubusercontent.com/dugword/Number-Bytes-Human/master/META.info
```
Comment (astj) [inline META.list]: Since your META file moved to META6.json, this line should be changed also.

**Comment by astj** [comment]
@dugword Nice work! I commented above, please check it.

**Comment by astj** [comment]
Thank you!

**Comment by dugword** [comment]
D'oh, that's right. Added a new commit to the pull request to reflect the name change to META6.json


Thanks astj

**Pull request merged**